### ISSUE TITLE
How does the classification loss function work?

### ISSUE THREAD

**Issue by MichaelMonashev** [fizyr/keras-maskrcnn#59]
Are images for classification considered after cropping the picture by bounding box and after applying a mask?

**Comment by vcarpani** [comment]
After cropping the bounding box, but masks do not play any roles in classification.
For this kind of question it would be better to use our slack channel #keras-maskrcnn
